frontend/locales/en/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "global": {
     "menu": {
-      "header": "ParallelCluster Manager",
+      "header": "AWS ParallelCluster UI",
```

Comment:
suggestion: let's create a key holding the name and re-use it, see [here](https://www.i18next.com/translation-function/nesting)

Reply:
Gotcha, thanks :)